src/yagolic.c

```diff
@@ -4,13 +4,13 @@
 #include "board.h"
 
 int main() {
-  int turns = 0;
+  unsigned long turns = 0;
```

Comment:
I don't think the game will ever exceed unsigned char tbh

Reply:
Fair enough.